onnxruntime/core/providers/cpu/generator/constant_of_shape.cc

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/framework/tensorutils.h"
+#include "core/providers/cpu/generator/constant_of_shape.h"
+#include "gsl/span"
+
+using namespace ::onnxruntime::common;
+using namespace ONNX_NAMESPACE;
+namespace onnxruntime {
+
+ONNX_CPU_OPERATOR_KERNEL(
+    ConstantOfShape,
+    9,
+    KernelDefBuilder()
+        .TypeConstraint("T1", DataTypeImpl::GetTensorType<int64_t>())
+        .TypeConstraint("T2", std::vector<MLDataType>{
+                                  DataTypeImpl::GetTensorType<MLFloat16>(),
```

Comment:
DataTypeImpl::AllFixedSizeTensorTypes() doesn't work?

Reply:
That one includes BFloat16 now which is not in the spec. Not sure which one needs to be corrected.